app/packs/_components/QuestionPack.tsx

```diff
@@ -147,7 +148,7 @@ export default function QuestionPack({
 						</div>
 						<div className="flex flex-col gap-0.5">
 							<h3 className="text-sm text-muted-foreground">Type</h3>
-							<p>{packMap[type]}</p>
+							<p>{packMap[type as PackType]}</p>
```

Comment:
We should update the interface on like 35 so that type is PackType, and that means down here we can remove the casting.
```suggestion
							<p>{packMap[type]}</p>
```